adding support to tif images

username77 opened the pull request:
add tif ccitt g4 to pdf because if we add as png/jpeg pdf increase the size 10x the size of tiff.
in this way we can load the tiff with BitMiracle.LibTiff and put into AddCcittG4

Commit:
adding support to tif images

Commit:
autoversion

username77 committed:
Merge branch 'UglyToad:master' into master

Commit:
Merge branch 'master' of https://github.com/username77/PdfPig

Commit:
removed assembly version

Commit:
removed generateonbuild

Commit:
cleaned

Commit:
addung tests

Commit:
CcittG4 tests

Commit:
tiff to pdf write test

Commit:
cleaned

Commit:
last clean

username77 commented:
try to see if is ok now

Commit:
removed blackis to avoid double inverse and added EndOfBlock for some tiffs

username77 committed:
Merge branch 'UglyToad:master' into master

username77 committed:
Merge upstream/master and preserve CCITT G4 writer tests

username77 committed:
Merge remote-tracking branch 'origin/master'

username77 commented:
I’ve updated my fork and synchronized this PR branch with the latest changes on my side. It should now be ready for another review when you have time. Thanks again!

username77 committed:
Merge branch 'UglyToad:master' into master

BobLd commented on src/UglyToad.PdfPig.Core/UglyToad.PdfPig.Core.csproj:
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Can you undo any change here? Git is flagging the file as changed

BobLd commented on src/UglyToad.PdfPig.DocumentLayoutAnalysis/UglyToad.PdfPig.DocumentLayoutAnalysis.csproj:
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
```
Can you undo any change here? Git is flagging the file as changed

BobLd commented on src/UglyToad.PdfPig.Fonts/UglyToad.PdfPig.Fonts.csproj:
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Can you undo any change here? Git is flagging the file as changed

BobLd commented on src/UglyToad.PdfPig.Tokenization/UglyToad.PdfPig.Tokenization.csproj:
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Can you undo any change here? Git is flagging the file as changed

BobLd commented on src/UglyToad.PdfPig.Tokens/UglyToad.PdfPig.Tokens.csproj:
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Can you undo any change here? Git is flagging the file as changed

username77 committed:
Remove LibTiff-based CCITT fixture generator

Drop the LibTiff-based fixture tool and TIFF source file, keep tests using the pre-generated CCITT payload and minimal metadata, and remove accidental BOM-only csproj changes flagged in review.
